### ISSUE TITLE
Complicated Model Implementation for Beer_MolBioSystems2014

### ISSUE THREAD

**Issue by FFroehlich** [Benchmarking-Initiative/Benchmark-Models-PEtab#114]
Isn't the whole purpose of PEtab not to have experiment specific parameters in the model?

**Comment by JanHasenauer** [comment]
indeed. the problem was that initially a nicer implementation of the model was not possible given the limitations of the toolboxes.
the implementation should definitely be updated.
